test/unit/url.test.js

```diff
@@ -951,6 +951,47 @@ describe('Url', function () {
             var url = new Url('http://127.0.0.1/hello/world/?query=param&query2=param2#test-api');
             expect(url.query.toObject()).to.eql({ query: 'param', query2: 'param2' });
         });
+
+        describe('with # in them', function () {
+            it('should ignore # if used for variable names', function () {
```

Comment:
Change the test title to read: `should not truncate variables at with # in their names`, the current content is misleading.